experimental/iterators/lib/Utils/RuntimeUtils.cpp

```diff
@@ -1,18 +1,51 @@
 //===- RuntimeUtils.cpp - Utils for MLIR / C++ interop -------------------===//
 
 #include "iterators/Utils/RuntimeUtils.h"
+#include "iterators/Operators/ColumnScanOperator.h"
+#include "iterators/Operators/ReduceOperator.h"
+#include "iterators/Utils/Tuple.h"
 
 #include <cstdio>
+#include <iostream>
+#include <tuple>
 
 //===---------------------------------------------------------------------===//
 // Exposed C API.
 //===---------------------------------------------------------------------===//
-extern "C" int64_t oneWay() {
-  void *p = reinterpret_cast<void *>(0xDEADBEEF);
-  fprintf(stdout, "Create a dummy pointer %p\n", p);
-  return reinterpret_cast<int64_t>(p);
+
+extern "C" int8_t *makeSampleInputOperator() {
+  std::vector<int32_t> sampleInput = {1, 2, 3};
+  using ScanOperatorType = decltype(makeColumnScanOperator(sampleInput));
+  auto const ptr =
+      new ScanOperatorType(sampleInput); // XXX(ingomueller) this is a leak!
```

Comment:
You def. want a `free` counterpart operation that you call explicitly from MLIR.
I see the reliance on `decltype` and the need to pass that type information across make/free boundaries.

For now I think I would just go with `makeSampleInputVectorInt32Operator` and `SampleInputVectorInt32Operator` and do the proper cast in the free.